cypress/integration/conditions/numeric_conditions.cy.js

```diff
@@ -11,17 +15,17 @@ import {
 } from '../../helpers/table.js'
 import { EXTENDED_TIMEOUT } from '../../support/util.js'
 
-const event = TEST_EVENT_DATA[1]
-const dimensionName = event.dimensions[0]
+const event = ANALYTICS_PROGRAM
```

Comment:
Is it needed to define new constants or can you just plug ANALYTICS_PROGRAM directly into where it is used (eg line 24)

Reply:
Good point, yeah I was also considering it. But I wasn't sure how dependent on the Analytics program we'd want to be. It's used in two places (see code below), so I figured a const was useful. But since everything is built on the Analytics program should maybe consider not even passing it around in the first place... Either way I'd leave that for a separate PR though.

```
const event = ANALYTICS_PROGRAM
const dimensionName = TEST_DIM_NUMBER
const periodLabel = event[DIMENSION_ID_EVENT_DATE]
const stageName = 'Stage 1 - Repeatable'

const setUpTable = () => {
    selectEventProgramDimensions({ ...event, dimensions: [dimensionName] })
```